backend/src/layer/data/flickr_api/test/const_test_data.rs

```diff
@@ -0,0 +1,51 @@
+use std::time::Duration;
+use std::{env, fs};
+use dotenvy::dotenv;
+use reqwest::Client;
+
+/// For all valid tests this image 'll be used: https://www.flickr.com/photos/gerdavs/52310534489/
+
+pub fn get_expected_get_size_result() -> String {
```

Comment:
consider using `include_str!` which loads the file at compile time. then a const variable should be sufficient.